deployment/node-feature-discovery/templates/nfd-worker-conf.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nfd-worker-conf
+  namespace: {{ .Release.Namespace }}
```

Comment:
Could you educate me on helm: is the namespace needed to be defined? If so, why?

Reply:
it does not i will remove, helm will set the namespace during installation